src/main/java/br/com/caelum/comidinhas/handler/validation/ValidationErrorsOutputDto.java

```diff
@@ -0,0 +1,30 @@
+package br.com.caelum.comidinhas.handler.validation;
+
+import java.util.*;
+
+public class ValidationErrorsOutputDto {
+
+    private List<String> globalErrorMessages = new ArrayList<>();
+    private List<FieldErrorOutputDto> fieldErrors = new ArrayList<>();
+
+    public void addError(String message) {
+        globalErrorMessages.add(message);
+    }
+
+    public void addFieldError(String field, String message) {
+        FieldErrorOutputDto fieldError = new FieldErrorOutputDto(field, message);
+        fieldErrors.add(fieldError);
+    }
```

Comment:
Esses métodos não precisam ser públicos mas os getters precisam.

```suggestion
    void addError(String message) {
        globalErrorMessages.add(message);
    }

    void addFieldError(String field, String message) {
        FieldErrorOutputDto fieldError = new FieldErrorOutputDto(field, message);
        fieldErrors.add(fieldError);
    }
```